utils/config/chain.config.ts

```diff
@@ -24,7 +24,7 @@ const DEFAULT_CHAIN_PROPERTIES: ChainProperties = toChainProperty(
 
 const chainPropertyMap: Config<ChainProperties> = {
   rmrk: DEFAULT_CHAIN_PROPERTIES,
-  bsx: toChainProperty(10041, 12, 'BSX'),
+  bsx: toChainProperty(10041, 12, 'BSX', 'https://basilisk.subscan.io/'),
```

Comment:
not ready as we still are on bsx testnet
This show "false" link in history component

Reply:
So maybe I still hide bsx link, only show ksm link, is this ok?

Reply:
yeah hide it for bsx for now until we go with mainnet
I think it was possible to see the transactions/blocks using polkdadotjs

Reply:
What's the url? Could u pls give a demo, thanks🙂

Reply:
https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc-01.basilisk.hydradx.io#/explorer

with custom node endpoint

Reply:
Thanks!
So do we still need to hide current bsx(testnet) or we use this endpoint to show something?

Reply:
hide bsx, I sense that polkadot js link will be hard to find/make

Reply:
Ok